bin/varnishncsa/varnishncsa.c

```diff
@@ -164,8 +164,26 @@ static struct ctx {
 	int			recd_beresp;
 } CTX;
 
+enum format_policy {
+        FMTPOL_FIRST = 0,
+        FMTPOL_ALWAYS
+};
```

Comment:
Let's populate the enum entirely right off the bat with all policies, even if we initially honor a subset of the rules.